docrawl/docrawl_core.py

```diff
@@ -820,7 +830,9 @@ def _extract_multiple_xpaths(self, inp):
         df.to_excel(short_filename + ".xlsx")
 
         with open(filename, "w+", encoding="utf-8") as f:
-            pass
+            output_list = ["\n".join(result_element) for result_element in result]
+            output = "\n".join(output_list)
+            f.write(output)
```

Comment:
this txt file is not supposed to hold any information - it was just to maintain threading - transfer data through dataframe above